src/test/java/com/hillel/items_exchange/exceptionshandling/GlobalExceptionHandlerTest.java

```diff
@@ -157,6 +160,15 @@ void testHandleMethodArgumentNotValidException() throws Exception {
         assertThat(result.getResolvedException(), is(instanceOf(MethodArgumentNotValidException.class)));
     }
 
+    @Test
+    void testHandleEntityAmountException() throws Exception {
+        doThrow(new EntityAmountException(MessageSourceUtil
```

Comment:
Why don't you allowed the code trowing such exception by creation List of 10 children?

Reply:
If I understood your comment correctly, if create List of 10 children -> throws ValidationException. This exception throws when, eq, user have several children and he tries to add more and this quantity more than 10.